cc_midterm/cc_midterm/models.py

```diff
@@ -32,21 +24,19 @@ class Products(models.Model):
     natural_organic_flag = models.CharField(db_column='NATURAL_ORGANIC_FLAG', max_length=10, blank=True, null=True)  # Field name made lowercase.
 
     class Meta:
-        managed = False
         db_table = 'products'
 
 
 class Transactions(models.Model):
     basket_num = models.CharField(db_column='BASKET_NUM', max_length=250, primary_key=True)  # Field name made lowercase.
-    hshd_num = models.SmallIntegerField(db_column='HSHD_NUM', blank=True, null=True)  # Field name made lowercase.
+    hshd_num = models.SmallIntegerField(Households, on_delete=models.CASCADE, db_column='HSHD_NUM', blank=True, null=True)  # Field name made lowercase.
```

Comment:
This throws an error when running `run-dev.sh`:
```
...
    hshd_num = models.SmallIntegerField(Households, on_delete=models.CASCADE, db_column='HSHD_NUM', blank=True, null=True)  # Field name made lowercase.
TypeError: __init__() got an unexpected keyword argument 'on_delete'
```

Reply:
I must have missed changing that SmallIntegerField to a ForeignKey when resolving merge conflicts. Should have the fix up in a moment.